database/migrations/2020_09_29_144334_create_categories_table.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateCategoriesTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('categories', function (Blueprint $table) {
+            $table->increments('id');
+            $table->integer('parent_id')->unsigned()->default(config('category_comment.parent'));
```

Comment:
sao em lại đặt tên là category_comment
```suggestion
            $table->integer('parent_id')->unsigned()->default(config('category.parent'));
```

Reply:
cái này chỉ để lưu vào config. vì e dùng nó luôn cả comment, mà dùng biến tĩnh thì thảo bảo cho vào config, e thấy tạo 2 cái file category và comment không cần thiết vì chỉ return 1 biến thôi ạ.

Reply:
với lại e không nghĩ được tên nào làm chung cho 2 đứa đành đặt tên đấy thôi ạ.